org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/messages.properties

```
@@ -23,6 +23,16 @@ XMLCatalogPreferencePage_OpenInEditorApplyAndEdit=Apply and Edit
 # XML CodeLens preference page
 XMLCodelensPreferencePage_codelens_enabled=Enable XML &CodeLens
 
+# XML Formatting preference page
+XMLFormattingPreferencePage_format_emptyElements=Expand/collapse empty elements
+XMLFormattingPreferencePage_format_spaceBeforeEmptyCloseTag=Insert space before end of self closing tag
+XMLFormattingPreferencePage_format_splitAttributes=&Split multiple attributes each onto a new line
+XMLFormattingPreferencePage_format_splitAttributesIndentSize=&Levels to indent the attributes
+XMLFormattingPreferencePage_format_closingBracketNewLine=Put a closing bracket on a newline
+XMLFormattingPreferencePage_format_preserveAttributeLineBreaks=&Preserve line breaks that appear before and after attributes
+XMLFormattingPreferencePage_format_xsiSchemaLocationSplit=Split `xsi:schemaLocation` content
```

Not sure what it means?
```suggestion
XMLFormattingPreferencePage_format_xsiSchemaLocationSplit=Place URLs in `xsi:schemaLocation` attribute on separate lines
```
?

---

See https://github.com/redhat-developer/vscode-xml/blob/main/docs/Formatting.md#xmlformatxsischemalocationsplit

---

I would like to keep this label since we use it too in vscode-xml.